src/services/dataStore.ts

```
@@ -0,0 +1,623 @@
+    );
+  }
+
+  static async encrypt(data: string): Promise<string> {
+    try {
+      const key = await this.getKey();
+      const iv = crypto.getRandomValues(new Uint8Array(12));
+      const encodedData = new TextEncoder().encode(data);
+      
+      const encrypted = await crypto.subtle.encrypt(
+        { name: "AES-GCM", iv },
+        key,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)